via/models/_mixins.py

```diff
@@ -0,0 +1,16 @@
+from datetime import datetime
+
+from sqlalchemy import func
+from sqlalchemy.orm import Mapped, MappedAsDataclass, mapped_column
+
+
+class CreatedUpdatedMixin(MappedAsDataclass):
+    created: Mapped[datetime] = mapped_column(
+        init=False,
+        server_default=func.now(),  # pylint:disable=not-callable
```

Comment:
I was holding the SQLA alchemy PR over LMS because this is annoying (not-callable)

I guess I'll have to bite the bullet and spirnkle a few of these there 
https://github.com/hypothesis/lms/pull/5283